### ISSUE TITLE
[VINICIUS] Levantar as ferramentas e as referências bibliográficas para o projeto

### ISSUE THREAD

**Issue by hyperiondeimos** [hyperiondeimos/PCCT2018#7]
O mês de março será utilizado para que vocês possam ter a liberdade de pesquisar e verificar as ferramentas necessárias para a construção de seu projeto. Aqui você pode também me informar sobre duvidas e até mesmo travas na pesquisa.

**Comment by MViniciusSC** [comment]
Introdução a utilização de GPIO do Raspberry: https://youtu.be/uevdLmyq1tU

Eu encontrei poucas fontes falando sobre leitores biométricos para Raspberry.
Esse é um projeto para ônibus usando o sensor biométrico, tá em espanhol mas da pra entender [Projeto](https://prezi.com/0332qgbqnxpi/fingerpi/)

Esse seria o modelo de fingerprint que eu usaria nesse caso: [Sensor biométrico](https://www.sparkfun.com/products/14585)

**Comment by hyperiondeimos** [comment]
Todos os sensores e dispositivos digitais que existem no Arduino funcionam plenamente no Raspberry sem problemas. Você não precisa somente programar em Python se assim desejar.

**Comment by MViniciusSC** [comment]
Ah, eu não sabia disso. Obrigado

**Comment by hyperiondeimos** [comment]
Claro, olha só esse sensor aqui para o [Arduino](https://produto.mercadolivre.com.br/MLB-934842669-modulo-sensor-leitor-biometrico-impresso-digital-arduino-_JM).
E um [tutorial](https://tutorials-raspberrypi.com/how-to-use-raspberry-pi-fingerprint-sensor-authentication/) em inglês que fala sobre como colocar ele funcionando.
